tests/ci_build/Dockerfile.cpu_clang

```diff
@@ -19,3 +19,9 @@ RUN wget -O - http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
     apt-get install -y clang-3.9 clang-5.0 && \
     clang-3.9 --version && \
     clang-5.0 --version
+
+# Add MKLML library, compatiable with Ubuntu16.04
+RUN wget --no-check-certificate -O /tmp/mklml.tgz https://github.com/01org/mkl-dnn/releases/download/v0.12/mklml_lnx_2018.0.1.20171227.tgz
```

Comment:
Please update all dockerfiles in that directory in order to use the correct mkl version